syntax = "proto3";

package syft.core.node.common.service;

import "proto/core/common/common_object.proto";
import "proto/core/io/address.proto";

message RemoteAddMessage {
  syft.core.common.UID msg_id = 1;
  syft.core.common.UID id_at_location = 2;
  syft.core.io.Address address = 3;
  int64 num = 4;
}
